modules/swagger-codegen/src/main/resources/php/configuration.mustache

```diff
@@ -110,7 +110,7 @@ class Configuration
      *
      * @var string
      */
-    protected $userAgent = "{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen/{{{artifactVersion}}}/php{{/httpUserAgent}}";
+    protected $userAgent = "{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen{{#artifactVersion}}/{{{.}}}{{/artifactVersion}}/php{{/httpUserAgent}}";
```

Comment:
I think we should put in something instead of the `artifactVersion` when it is not set.

I think I read something about the default user agent is intended to follow a format of `Swagger-Codegen/version/language` (but I can't find the reference for that now).

So maybe we should just put in the string "unknown-version" when we don't have a better guess.